rules/linux/credential_access_potential_linux_ssh_bruteforce_external.toml

```diff
@@ -0,0 +1,89 @@
+a short time interval. Adversaries will often brute force login attempts across multiple users with a common or known
+password, in an attempt to gain access to these accounts.
+"""
+from = "now-9m"
+index = ["logs-system.auth-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential External Linux SSH Brute Force Detected"
+note = """## Triage and analysis
+
+### Investigating Potential External Linux SSH Brute Force Detected
+
+The rule identifies consecutive SSH login failures targeting a user account from the same source IP address to the same target host indicating brute force login attempts.
+
+This rule will generate a lot of noise for systems with a front-facing SSH service. 
+Adversaries scan the whole internet for remotely accessible SSH services, and try to brute force them in order to gain unauthorized access. 
+In case this rule generates too much noise, and external brute forcing is of not much interest, consider turning this rule off and enabling "Potential Internal Linux SSH Brute Force Detected" to detect internal brute force attempts.
+
+#### Possible investigation steps
+
+- Investigate the login failure user name(s).
+- Investigate the source IP address of the failed ssh login attempt(s).
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Identify the source and the target computer and their roles in the IT environment.
+
+### False positive analysis
+
+- Authentication misconfiguration or obsolete credentials.
+- Service account password expired.
+- Infrastructure or availability issue.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved hosts to prevent further post-compromise behavior.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are identified. Reset passwords for these accounts and other potentially compromised credentials, such as email, business systems, and web services.
+- Run a full antimalware scan. This may reveal additional artifacts left in the system, persistence mechanisms, and malware components.
+- Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
+- Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
+
+"""
+risk_score = 21
+rule_id = "fa210b61-b627-4e5e-86f4-17e8270656ab"
+severity = "low"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Credential Access"]
+type = "eql"
+query = '''
+sequence by host.id, source.ip, user.name with maxspan=5s
+  [ authentication where host.os.type == "linux" and 
+   event.action in ("ssh_login", "user_login") and event.outcome == "failure" and
+   not cidrmatch(source.ip, "10.0.0.0/8", "127.0.0.0/8", "169.254.0.0/16", "172.16.0.0/12", "192.0.0.0/24") ] with runs = 3
```

Comment:
This one was resolved but received no comments
https://github.com/elastic/detection-rules/pull/2583#discussion_r1142037224

Reply:
I added your suggestion to the PR, that's why I resolved it. cidr matching on destination IP doesn't make sense right? So I just took the IP ranges from your suggestion (because I was indeed missing several private / loopback ranges) and added it to the query.

Reply:
Fixed!